.pre-commit-config.yaml

```diff
@@ -107,7 +107,7 @@ repos:
         exclude: "(^docs/source/_templates/)"
         args:
           [
-            "--ignore-directives=automodule",
```

Comment:
the ignore of automodule is removed? Does it means we will never use automodule in this repo?

Reply:
because wu use autoapi to replace "autoclass", "automobile"